### DISCUSSION TITLE
Adding a paragraph or guide about private keys/seed phrase

### DISCUSSION THREAD

**Discussion by ObjectifMoon21** [sovenor/bitcoin-rocks#21] [Ideas]
Hi @sovenor After reviewing the wallets guide, I believe a paragraph should be added to explain how private and public keys works and why it’s crucial to keep private keys offline. 

The guide should also educate on the types of seed phrases (12 or 24 words), passphrases, and methods for storing them on metal plates or other waterproof, fireproof storage with video or reviews like this one https://jlopp.github.io/metal-bitcoin-storage-reviews/

New users should be informed that if they are unsure about how to use and store private keys, rather than lose their bitcoin without knowing what they're doing, they should start with a custodial wallet that has a strong password + 2FA. This approach allows them to begin using Bitcoin safely before learning more and transitioning to self-custody.

Let me know if you don't have time to do it anytime soon I can start writing one these days and send it for review.

**Comment by sovenor** [comment]
Great idea, I definitely agree this would be an improvement. I'll work on the specific text for it soon.

**Comment by sovenor** [comment]
Closing this discussion and moving it to issue #27. I will work on this ASAP.
